src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -30,6 +31,6 @@ void npe2(String a, String b) {
 
  @Override
  public boolean equals(Object obj) {
-  return true;
+  return false;
```

Comment:
**Reporter**: PMD
**Rule**: OverrideBothEqualsAndHashcode
**Severity**: WARN
**File**: src/main/java/se/bjurr/violations/lib/example/MyClass.java L34

Basic http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/basic.html#OverrideBothEqualsAndHashcode

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<59982958>*